src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,64 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
+    public static final String SUPPLY_FIELD = "supply";
+    public static final String BUY_FIELD = "buy";
+    public static final String RESULT_FIELD = "result";
+    public static final String SEPARATOR = ",";
+    public static final String INITIAL_AMOUNT = "0";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        File file = new File(fromFileName);
+        List<String> originalReport = readReport(file);
+        String[][] finishedReport = getStatsArray(originalReport);
+        file = new File(toFileName);
+        writeReport(file, finishedReport);
```

Comment:
Please move creation of variable file into method it will be used in
```suggestion
        List<String> fileContent = readFile(fromFileName);
        String report = generateReport(fileContent);
        writeTofile(toFileName, report);
```